Users/vivasa/symboltask/node10warning (#20140)
microsoft/azure-pipelines-tasks: Tasks/PublishSymbolsV2/README.md

# Symbol Publish

### Overview

The task is used to publish symbols. 

### Contact Information

> Please report a problem at [Stackoverflow](https://stackoverflow.microsoft.com/posts/tagged/20485) if you are facing problems in making this task work.  You can also share feedback about the task like, what more functionality should be added to the task, what other tasks you would like to have, at the same place.

Reviewer: not a public url

> Please report a problem at [Github Repo Issues](https://github.com/microsoft/azure-pipelines-tasks/issues) if you are facing problems in making this task work.  You can also share feedback about the task like, what more functionality should be added to the task, what other tasks you would like to have, at the same place.

Author: Updated

### Prerequisite for the task

**PowerShell**

> The task needs at least version 3.0 of Powershell on the target machine.

Reviewer: ```suggestion
The task requires Powershell version 3.0 or greater.
```

**TypeScript**

> The task needs at least node16/node20 agent on the target machine. Node10 support is going to stop on 07/31/2024.

Reviewer: ```suggestion
The task requires either Node v16 or Node v20 to run. Node v10 support is no longer supported, starting 07/31/2024.
```

* **PublishSymbols**: Indicates whether to publish the symbol files.

  * **SymbolServerType**: Choose where to publish symbols. Symbols published to the Azure Artifacts symbol server are accessible by any user with access to the organization/collection. Azure DevOps Server only supports the \"File share\" option. Follow [these instructions](https://go.microsoft.com/fwlink/?linkid=846265) to use Symbol Server in Azure Artifacts. Effective when "PublishSymbols = true"

  * **SymbolsPath**: The file share that hosts your symbols. This value will be used in the call to `symstore.exe add` as the `/s` parameter.. Effective when "PublishSymbols = true && SymbolServerType = FileShare"

  * **CompressSymbols**: Compress symbols when publishing to file share. Effective when "SymbolServerType = FileShare"

  * **SymbolExpirationInDays**: The number of days that symbols should be retained. Effective when "PublishSymbols = true && SymbolServerType = TeamServices"

  * **IndexableFileFormats**: Which debug formats to publish to the symbol server. Effective when "PublishSymbols = true && SymbolServerType = TeamServices"

> 

Reviewer: where did the bullet points go?
